docs/tasks/active/20260423-cell-structural-edits-todo.md

```
@@ -0,0 +1,404 @@
+---
+
+### Task 5: Mark Phase 8 shipped in design doc
+
+**Files:**
+- Modify: `docs/design/docs/docs-intent-preserving-edits.md`
+
+- [ ] **Step 1: Update Phase 8 status**
+
+In `docs/design/docs/docs-intent-preserving-edits.md`, change:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)